spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+RSpec.describe 'User show', type: :feature do
+  before(:each) do
+    # rubocop: disable Layout/LineLength
+
+    @first_user = User.create(name: 'Tom',
+                              photo: 'https://www.thoughtco.com/thmb/0I-Uw-0rcc6MUzcZJauNGKR9JzA=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/male-computer-programmer-using-laptop-at-desk-in-office-755650739-5c5bb32346e0fb0001f24d3d.jpg', bio: 'Teacher from Mexico.', posts_counter: 0)
+    # rubocop: enable Layout/LineLength
+
+    @first_post = Post.create(author: @first_user, title: 'Hello', text: 'This is my first post', comments_counter: 0,
+                              likes_counter: 0)
+    @second_post = Post.create(author: @first_user, title: 'Hola', text: 'This is my second post', comments_counter: 0,
+                               likes_counter: 0)
+    @third_post = Post.create(author: @first_user, title: 'Honjour', text: 'This is my third post',
+                              comments_counter: 0, likes_counter: 0)
+    @fourth_post = Post.create(author: @first_user, title: 'Salam', text: 'This is my fourth post',
+                               comments_counter: 0, likes_counter: 0)
+
+    visit user_path(@first_user)
+  end
+
+  describe 'user show page' do
+    it 'shows the user profile picture' do
+      expect(page).to have_xpath("//img[contains(@src,'https://www.thoughtco.com/thmb/0I-Uw-0rcc6MUzcZJauNGKR9JzA=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/male-computer-programmer-using-laptop-at-desk-in-office-755650739-5c5bb32346e0fb0001f24d3d.jpg')]")
+    end
+
+    it 'shows the username' do
```

Comment:
> * [ ]  **[OPTIONAL]** It is not a good practice to embed a `describe` block within another `describe` block. I would recommend you move the inner one out of this one. If you wish to provide more description within a describe block you can use the `context` block instead.
> * [ ]  **[OPTIONAL]** I think this second test should be **post show page**, as the index page is handled in the previous file

- [OPTIONAL] As requested by the previous, kindly reviewer, make sure to address this 💯 👍🏻